tasks/libs/common/gitlab.py

```diff
@@ -138,16 +147,18 @@ def jobs(self, project_name, pipeline_id, page=1, per_page=100):
         path = "/projects/{}/pipelines/{}/jobs?per_page={}&page={}".format(
             quote(project_name, safe=""), pipeline_id, per_page, page
         )
-        return self.make_request(path, json=True)
+        return self.make_request(path, json_output=True)
 
     def find_tag(self, project_name, tag_name):
         """
         Look up a tag by its name.
         """
         path = "/projects/{}/repository/tags/{}".format(quote(project_name, safe=""), tag_name)
-        return self.make_request(path, json=True)
+        return self.make_request(path, json_output=True)
 
-    def make_request(self, path, headers=None, data=None, json=False, stream=False, method=None):
+    def make_request(
+        self, path, headers=None, data=None, json_input=False, json_output=False, stream_output=False, method=None
```

Comment:
I think we should start documenting the behavior of fields here since there are quite a lot of them (can be done on a different PR)